src/main/scala/is/hail/expr/ir/functions/MathFunctions.scala

```diff
@@ -48,13 +62,17 @@ object MathFunctions extends RegistryFunctions {
     registerIR("toFloat32", tnum("T"))(x => Cast(x, TFloat32()))
     registerIR("toFloat64", tnum("T"))(x => Cast(x, TFloat64()))
 
+    registerScalaFunction("**", TInt32(), TInt32(), TFloat64())(thisClass, "pow")
+    registerScalaFunction("**", TInt64(), TInt64(), TFloat64())(thisClass, "pow")
+    registerScalaFunction("**", TFloat32(), TFloat32(), TFloat64())(thisClass, "pow")
+    registerScalaFunction("**", TFloat64(), TFloat64(), TFloat64())(thisClass, "pow")
+
     registerScalaFunction("exp", TFloat64(), TFloat64())(mathPackageClass, "exp")
     registerScalaFunction("log10", TFloat64(), TFloat64())(mathPackageClass, "log10")
     registerScalaFunction("sqrt", TFloat64(), TFloat64())(mathPackageClass, "sqrt")
     registerScalaFunction("log", TFloat64(), TFloat64())(mathPackageClass, "log")
     registerScalaFunction("log", TFloat64(), TFloat64(), TFloat64())(thisClass, "log")
     registerScalaFunction("pow", TFloat64(), TFloat64(), TFloat64())(mathPackageClass, "pow")
```

Comment:
"pow" isn't used anymore. I think you can delete it both here and in the function registry.